ext/hal/ti/simplelink/README

```diff
@@ -22,7 +22,8 @@ The driver library source is copied from the SDK, as follows:
   popd
 
 The source file startup_files/ccfg.c has been modified to use the appropriate
-sections when linking.
+sections when linking. In addition, the file inc/hw_flash.h has been modified
```

Comment:
We should handle this by fixing/update the compliance script.

Reply:
I dropped that commit and will fix it in the compliance script. I originally had PR zephyrproject-rtos/ci-tools#52 to add CONFIG_CRC to the whitelist, but when I discovered it was just in a comment I took this approach instead. I can rebase and reopen that. It seems like the right solution might be for the tool to ignore comments.